test/contracts-network/colony-network-auction.js

```diff
@@ -63,6 +65,23 @@ contract("Colony Network Auction", accounts => {
       await checkErrorRevert(colonyNetwork.startTokenAuction(ethers.constants.AddressZero), "colony-auction-invalid-token");
     });
 
+    it("should fail with a zero clny token", async () => {
+      const metaColonyVersion = await metaColony.version();
+      const metaColonyResolverAddress = await colonyNetwork.getColonyVersionResolver(metaColonyVersion);
+      const metaColonyResolver = await Resolver.at(metaColonyResolverAddress);
+      const contractEditing = await ContractEditing.new();
+      await metaColonyResolver.register("setStorageSlot(uint256,bytes32)", contractEditing.address);
```

Comment:
This block of code appears three times in these new tests, and perhaps elsewhere besides. Can we wrap this into a helper and re-use it?

Reply:
Done